mono/mini/method-to-ir.c

```diff
@@ -68,6 +68,7 @@
 #include <mono/utils/mono-threads-coop.h>
 #include <mono/utils/mono-utils-debug.h>
 #include <mono/utils/mono-logger-internals.h>
+#include <mono/metadata/verify-internals.h>
```

Comment:
I'm sorta in the fence on exposing the verifier to the JIT more than we need to.
Is this due to the header shenanigans you added there?

Reply:
It was to reuse the macros for token to index, token to  table -- token to fielddef.

Reply:
It was just  for IS_FIELD_DEF I think.